build/Tasks/BuildTask.cs

```diff
@@ -138,7 +139,7 @@ void RenderLine(string format, params object[] args)
 
         private void GenerateDocumentation(BuildContext context)
         {
-            context.DotNetCoreRun("Textrude", @"render --models ScriptLibrary/doc.yaml --template ScriptLibrary/doctemplate.sbn --output doc/lib.md");
+            context.DotNetCoreRun("src/Textrude", @"render --models src/ScriptLibrary/doc.yaml --template src/ScriptLibrary/doctemplate.sbn --output doc/lib.md");
         }
```

Comment:
Just a warning that you may see conflicts here when you rebase since I modified the GenerateDocumentation method.  Looks like the resolution should be quite straight forward though

Reply:
Ah ok, thanks for the heads-up 👍

Reply:
Looks like you're in the middle of a pretty heavy reorg !  I'll hold off committing any more changes for a bit

Reply:
Appreciated, but don't force holding of changing stuff around. I currently just moved the projects / solution into its own directory (src/) and I don't plan to change anything more drastic in this PR
(I am thinking about renaming some projects, but this would be a new PR)